data/src/main/java/com/linkedin/data/codec/AbstractJacksonDataCodec.java

```diff
@@ -213,12 +224,35 @@ public DataList readList(InputStream in) throws IOException
     return parse(_factory.createParser(in), mesg, locationMap);
   }
 
-  public void objectToJsonGenerator(Object object, JsonGenerator generator) throws IOException
```

Comment:
Do not change public APIs. Keep this one and add the second API

Reply:
Good catch. Fail to notice it was public